documentation/asciidoc/topics/ref_infinispan_config_logging.adoc

```diff
@@ -0,0 +1,14 @@
+[id='infinispan-logging-configuration_{context}']
+= {brandname} logging configuration
+
+{brandname} uses Apache Log4j to provide configurable logging mechanisms that capture details about the environment and record cache operations for troubleshooting purposes and root cause analysis.
```

Comment:
I'm not sure we need to explain the advantage of logging here, just that's it's possible to provide a full log4j logging configuration if required.

Maybe we should point the user to the `spec.logging` configuration in the CR as the preferred method and explain this approach should be used if advantage logging options, such as file-based logging are required?

e.g https://github.com/infinispan/infinispan-operator/issues/1275